client/components/LearnerViewDirective.js

```diff
@@ -52,6 +54,15 @@ tie.directive('learnerView', [function() {
                     </span>
                   </p>
                 </div>
+                <div class="tie-feedback-syntax-error">
+                <a href class="tie-feedback-syntax-error-link", ng-click="toggleSyntaxErrorHint()", 
+                        ng-show="isSyntaxErrorLinkShown">
```

Comment:
I think a better name might be something like "syntaxErrorFound" or "syntaxErrorExists".